### PR TITLE
Optimizely styling and bug fixes

### PR DESCRIPTION
**Pull request by davidfateh**
This work sharpens up the Optimizely App styling and also addresses some various bug fixes. I will mark the bug fixes in the code with comments.

Screenshot 2019-08-14 at 08 42 46

### CODE REVIEW

File: marketplace/optimizely/src/AppPage/ContentTypes.js [hunk 1]
```diff
@@ -63,18 +66,19 @@ export default function ContentTypes({
...
   onDeleteContentType
 }) {
-  const addableContentTypes = allContentTypes.filter(ct =>
-    isContentTypeValidSelection(ct, addedContentTypes)
-  );
-
   const [selectedContentType, selectContentType] = useState('');
   const [selectedReferenceFields, selectRef] = useState({});
   const [modalOpen, toggleModal] = useState(false);
+  const [isEditMode, setEditMode] = useState(false);
```
Comment (davidfateh) [inline marketplace/optimizely/src/AppPage/ContentTypes.js]: This logic allows for edit mode which disabled the select box box in the modal when only editing one content type.

File: marketplace/optimizely/src/optimizely-client.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ export default class OptimizelyClient {
       }
     });
 
-    if (response.status === 200) {
+    if (response.ok) {
```
Comment (davidfateh) [inline marketplace/optimizely/src/optimizely-client.js]: This fixes a bug where Optimizely sometimes gives us 2XX statuses, not just 200.

File: marketplace/optimizely/src/AppPage/index.js [hunk 3]
```diff
@@ -4,17 +4,32 @@ import tokens from '@contentful/forma-36-tokens';
 import { css } from 'emotion';
 import { Heading, Typography, Paragraph } from '@contentful/forma-36-react-components';
 
-import Features from './Features';
```
Comment (davidfateh) [inline marketplace/optimizely/src/AppPage/index.js]: No longer listing what the features are. That happens on the details modal.

File: marketplace/optimizely/src/AppPage/index.js [hunk 4]
```diff
@@ -26,6 +41,12 @@ const styles = {
   light: css({
     opacity: '0.6',
     marginTop: tokens.spacingM
+  }),
+  logo: css({
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'center',
+    marginTop: tokens.spacingXl
```
Comment (jelz) [inline marketplace/optimizely/src/AppPage/index.js]: I guess `marginBottom` would also be nice so the logo doesn't touch the browser window if more CTs are selected.

**Pull request merged**